docs/overrides.md

```diff
@@ -24,7 +24,7 @@ Files:
 
 **Decidim:Proposals:Proposals::show.html.erb**
 
-The `strip_tags` parameter is changed to `"false"`. This change fix the links that are not showing in the body.
+The `strip_tags` parameter is changed to `false` by overriding `safe_content?` and making it always `true`. This change allows all links to show in the body.
```

Comment:
saying that all content is safe is not very safe... don't we have another workaround? what was the problem with the previous implementation? I think there was a content_processor that converted to  external links

Also the title  of this block does not correspond to the override, but lets wait until we have a definitive implementation.

Reply:
The previous implementation would do the same, either admit or deny all links. Since in this version links are already being shown, the override is no longer needed.